<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>348</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Počet skenů</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Počet skenů v0.2.0-alpha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Spočítat skeny z vybraného adresáře; případně i další soubory a množství PSP</string>
    </property>
    <property name="text">
     <string>Spočítej (skeny)</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>85</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Vybrat adresář</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_start">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Skeny vytvořené od</string>
    </property>
    <property name="date">
     <date>
      <year>2025</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_end">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Skeny vytvořené do</string>
    </property>
    <property name="date">
     <date>
      <year>2025</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>84</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Datum od</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>113</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Datum do</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Vyberte adresář klinutím na tlačítko se 3 tečkami</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <property name="text">
     <string>Není vybraný žádný adresář</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>157</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vybraný adresář:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>v0.2.0-alpha</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_countTxtFiles">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Spočítat soubory s koncovkou txt</string>
    </property>
    <property name="text">
     <string>.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spočítat další typy souborů:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_countXmlFiles">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Spočítat soubory s koncovkou xml</string>
    </property>
    <property name="text">
     <string>.xml</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_countJp2Files">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Spočítat soubory s koncovkou jp2</string>
    </property>
    <property name="text">
     <string>.jp2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_countMd5Files">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Spočítat soubory s koncovkou md5</string>
    </property>
    <property name="text">
     <string>.md5</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_prefix">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Zadat kód knihovny (začátek názvu PSP - adresáře)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_countDirs">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Spočítat PSP (adresáře)</string>
    </property>
    <property name="text">
     <string>PSP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zadat kód knihovny (URN:NBN) a spočítat balíčky:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_generateLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Vygenerovat log po sečtení</string>
    </property>
    <property name="text">
     <string>Vygenerovat log</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
